torch/_inductor/mkldnn_lowerings.py

```diff
@@ -791,11 +818,11 @@ def inner_fn_requant(index, scale, zero_point):
                 )
             assert packed_weight.get_name() in V.graph.constants
             input_gen_fns = {
-                3: lambda x: V.graph.constants[x.get_name()],
-                4: lambda x: V.graph.constants[x.get_name()],
-                5: lambda x: V.graph.constants[x.get_name()],
-                6: lambda x: V.graph.constants[x.get_name()],  # For bias
+                3: lambda x: V.graph.constants[x.get_name()],  # packed weight
+                4: lambda x: V.graph.constants[x.get_name()],  # weight scale
+                6: lambda x: V.graph.constants[x.get_name()],  # bias
```

Comment:
Why do we need to change it? Can you remove the lines of unnecessary changes?

Reply:
w_zp is not being used in the computation, so I had removed it, since it doesn't have any side-effects.
 But I restored the line, following your comment

Reply:
@leslie-fang-intel, when we make `x_zp` a constant, should we also add it here?
How about `x_scale` if it's constant (static quantization)? Thanks!

Reply:
After offline discussion with @leslie-fang-intel, I added this change for `x_scale` & `x_zp`, whenever they'd be constant.